Dockerfile

```diff
@@ -1,12 +1,9 @@
-FROM python:3.8-slim
-COPY requirements.txt requirements.txt
```

Comment:
Same thing that I commented [here](https://github.com/codacy/codacy-checkov/pull/17#discussion_r732147962).
Copying all the things together makes every change to the source files trigger the download of the dependencies.

Reply:
Hum... I was just thinking about CI. I see your point if thinking from a developer perspective which is building images locally to test source code changes. You could change the Dockerfile for local development, but that's a hassle. A couple of layers shouldn't add much bulk and it makes your life easier. I'll change it.